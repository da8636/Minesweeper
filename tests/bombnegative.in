g 10 10
b 1 1
b -1 1
