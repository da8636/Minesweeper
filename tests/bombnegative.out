g 10 10
b 1 1
error
